bycycle/tests/group/test_epochs.py

```diff
@@ -0,0 +1,60 @@
+"""Test functions to compute features across epoched data."""
+
+import time
+
+import numpy as np
+
+import pytest
+
+from bycycle.group.epochs import compute_features_2d
+
+###################################################################################################
+###################################################################################################
+
+def test_compute_features_2d(sim_args):
+
+    sigs = np.array([sim_args['sig']] * 50)
+    fs  = sim_args['fs']
+    f_range = sim_args['f_range']
+
+    # Test returning only features, without samples
+    features = compute_features_2d(sigs, fs, f_range, n_cpus=1,
+                                   compute_features_kwargs={'return_samples': False})
+
+    for df_features in features:
+        assert df_features.equals(features[0])
+
+    # Sequential processing check
+    start = time.time()
+    features_seq, samples_seq = \
+        compute_features_2d(sigs, fs, f_range, n_cpus=1,
+                            compute_features_kwargs={'return_samples': True})
+    end = time.time()
+    elapsed_time_sequential = end - start
+
+    # Parallel processing check
+    start = time.time()
+    features, samples = compute_features_2d(sigs, fs, f_range, n_cpus=-1,
+                                            compute_features_kwargs={'return_samples': True})
+    end = time.time()
+    elapsed_time_parallel = end - start
+
+    # Since the same signal is used, check that each df is the same
+    for df_features in features_seq:
+        assert df_features.equals(features_seq[0])
+    for df_features in features:
+        assert df_features.equals(features[0])
+    for df_samples in samples_seq:
+        assert df_samples.equals(samples_seq[0])
+    for df_samples in samples:
+        assert df_samples.equals(samples[0])
+
+    # Assert that two cpus computes faster than one
+    assert elapsed_time_parallel < elapsed_time_sequential
```

Comment:
I'm not so sure about doing the whole timing thing, and testing it. 
Couple reasons:
- this feels to me like profiling, and not really testing
- parallel code is not guaranteed to be faster (sometimes the overhead exceeds gains), and this feels like something that could occur with our small test signals - meaning I wouldn't be surprised if in some possible cases the time check failed, but that would not be an informative / useful failure. 

I think then I would vote to not do the timing checks in these tests. What do you think?

Reply:
I'll remove the time tests. I agree with the second point, I had to increase the size of the array so it would pass.